src/plone/restapi/tests/test_field_serializer.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from plone.app.contenttypes.behaviors.richtext import IRichText
+from plone.dexterity.utils import createContentInContainer
+from plone.dexterity.utils import iterSchemata
+from plone.restapi.dxfields import DateFieldSerializer
+from plone.restapi.dxfields import DateTimeFieldSerializer
+from plone.restapi.dxfields import DefaultFieldSerializer
+from plone.restapi.interfaces import IFieldSerializer
+from plone.restapi.testing import PLONE_RESTAPI_FUNCTIONAL_TESTING
+from unittest2 import TestCase
+from zope.component import getMultiAdapter
+from zope.interface.verify import verifyClass
+from zope.schema import getFields
+import json
+
+
+class TestDexterityFieldSerializing(TestCase):
+    layer = PLONE_RESTAPI_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.request = self.layer['request']
+
+    def test_boolean(self):
+        folder = createContentInContainer(self.portal, u'Folder',
+                                          exclude_from_nav=True)
+        value = self.serialize(folder, u'exclude_from_nav')
+        self.assertEquals(True, value)
+        self.assertEquals(value, json.loads(json.dumps(value)))
+
+    def test_textline(self):
+        folder = createContentInContainer(self.portal, u'Folder',
+                                          title=u'My Folder')
+        value = self.serialize(folder, u'title')
+        self.assertEquals(u'My Folder', value)
+        self.assertEquals(value, json.loads(json.dumps(value)))
+
+    def test_choice(self):
+        folder = createContentInContainer(self.portal, u'Folder',
+                                          language=u'lo')
+        value = self.serialize(folder, u'language')
+        self.assertEquals(u'lo', value)
+        self.assertEquals(value, json.loads(json.dumps(value)))
+
+    def test_date(self):
+        folder = createContentInContainer(
+            self.portal, u'Folder',
+            effective_date=datetime(2017, 10, 22, 19, 45, 1))
+        value = self.serialize(folder, u'effective')
+        self.assertEquals(u'2017-10-22T19:45:01', value)
+        self.assertEquals(value, json.loads(json.dumps(value)))
+
+    def test_richtext(self):
+        document = createContentInContainer(
+            self.portal, u'Document',
+            text=IRichText['text'].fromUnicode(u'<p>Some Text</p>'))
+        value = self.serialize(document, u'text')
+        self.assertEquals({'raw': u'<p>Some Text</p>',
+                           'mimeType': 'text/html',
+                           'outputMimeType': 'text/x-html-safe',
+                           'encoding': 'utf-8'}, value)
```

Comment:
I think we should go with 3., which is:
```python
{
  "text": "<p>If you're seeing this instead of the web site you were expecting, the owner of this web site has just installed Plone. Do not contact the Plone Team or the Plone mailing lists about this.</p>",
}
```

Discussion:
- `encoding` is not necessary, since the whole document is encoded with the same encoding (`utf8`).
- `raw` is internal, the client does not nedd to know *what* we store
- `mimeType` is internal, the client does not need to know *how* we store it
- `outputMimeType` should be defined in the schema (e.g. `text` is always `text/html`)
- we therefore only require the actual transformed value.